app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from typing import List
+
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
```

Comment:
The type hint `List[Player]` assumes that `Player` is a class or type. Ensure that `Player` is correctly defined in `app.players.player` and is intended to be used as a type hint here.